core/src/main/resources/jenkins/model/Jenkins/systemInfo.jelly

```diff
@@ -29,6 +29,7 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <l:layout permissions="${app.MANAGE_AND_SYSTEM_READ}" title="${%System Information}">
     <st:include page="sidepanel.jelly" />
+    <l:breadcrumb href="" title="${%System Information}"/>
```

Comment:
Why `href`? Jellydoc says it can be omitted.

Reply:
```suggestion
    <l:breadcrumb title="${%System Information}"/>
```